MapleServer2/Servers/Game/GameSession.cs

```diff
@@ -61,7 +61,7 @@ protected override void EndSession(bool logoutNotice)
         FieldManager.RemovePlayer(Player);
         GameServer.PlayerManager.RemovePlayer(Player);
 
-        Player.OnlineCTS.Cancel();
```

Comment:
Just by creating a new character and not logging in can cause this line to reach? I don't think that's possible, since creating a character uses LoginSession 🤔

Reply:
That OnlineCTS NullReferenceError is from disconnecting game (In login) after creating new character.

I forgot `when playing in character and` prefix. 😭

Reply:
Sounds good!